mozci/console/commands/check_backfills.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+import os
+from collections import namedtuple
+from itertools import groupby
+
+import requests
+from cleo import Command
+from loguru import logger
+
+from mozci import config
+from mozci.push import make_push_objects
+from mozci.util.taskcluster import (
+    COMMUNITY_TASKCLUSTER_ROOT_URL,
+    find_task_id,
+    index_current_task,
+    list_dependent_tasks,
+    list_indexed_tasks,
+    notify_matrix,
+)
+
+BackfillTask = namedtuple("BackfillTask", ["task_id", "th_symbol", "state"])
+
+NOTIFICATION_BACKFILL_GROUP_COMPLETED = "Backfill tasks associated to the Treeherder symbol {th_symbol} for push {push.branch}/{push.rev} are all completed."
```

Comment:
@Archaeopteryx does this format for the notification look good to you? Can you suggest improvements if you think some are needed?

Reply:
Looks good to me.